e2e/app/page/admin/user-admin-table-page.ts

```diff
@@ -85,15 +83,15 @@ export default class UserAdminTablePage extends AuthenticatedPage {
   //extract only the Username text to verify the search box result
   async getUsernameText(rowIndex = 1, colIndex = 1): Promise<string> {
     const dataTable = this.getUserAdminTable();
-    const bodyTable = dataTable.getBodyTable();
+    const bodyTable = dataTable.getTable();
     const cell = await bodyTable.getCell(rowIndex, colIndex);
-    return getPropValue<string>(cell, 'textContent');
+    return getPropValue<string>(cell, 'innerText');
```

Comment:
TextContent includes all visible and invisible text within a tag. With PrimeReact the header value is now included in the body cells as invisible text.